airqo-platform/AirQo-api
# Ft locate redisgn

**_WHAT DOES THIS PR DO?_**
   - PR contains modifications in the locate microservice that enables recommendation for sensor placement utelising the smallest admin level for each country instead of restricting it to Parishes which are mainly in Uganda.
**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - []()
- GitHub issues
    - Closes #<enter GitHub issue number here>

**_HOW DO I TEST OUT THIS PR?_**
- Fetch branch locally
- cd to the src/locate directory
- activate the virtual environment
- install packages incase you're creating virtual environment for the first time
-  Ensure you have local db setup[staging access was disabled]
- run the flask app 
-  hit the endpoint  below [check for sample in postman] /or relatedly run through the frontend interface for sensor placement

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 
- http://127.0.0.1:4001/api/v1/locate/map/recommendations?tenant=airqo


**_ARE THERE ANY RELATED PRs?_**
